packages/fluent-theme/src/fluent/styles/SpinButton.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { fluentBorderRadius } from './styleConstants';
+import { NeutralColors } from '../FluentColors';
+
+const buttonStyles = {
+  color: NeutralColors.gray130,
+  width: 23,
```

Comment:
Is this `width` change intentional? Feels like this _could_ break existing SpinButton users. On the demo site, I'm seeing a slight difference (not the space to the left of 0):

**Default**
![image](https://user-images.githubusercontent.com/307118/47729684-dc1f4800-dc1d-11e8-90b2-42804cba3e96.png)

**Fluent**
![image](https://user-images.githubusercontent.com/307118/47729712-e8a3a080-dc1d-11e8-9167-1dd67f40eaf8.png)

Reply:
Agree with you on the randomness of the number. I followed the numbers in the toolkit and was planning to follow up with @betrue-final-final today.

Reply:
I talked to Christopher and yes the chevron is offset by 1px to the left. The reason for that is that it accounts for the right side border, making it 8px paddingleft, and effective 8px padding right to the end of the control.

Reply:
The borders look weird though.

Reply:
Borders look like this probably for two reasons:

1. Examples need space in between them. A regular `<br />` should fix the issue.
2. I believe the screenshots above were made after zooming in the browser which sometimes gives you these visual regressions as seen here:
![screen shot 2018-10-30 at 11 43 44 am](https://user-images.githubusercontent.com/29514833/47742073-15b17c80-dc39-11e8-94d1-965d9a2ba10f.png)

Reply:
@Jahnp Confirmed with the design that this is an intentional change.